R/html_dependency.R

```diff
@@ -365,6 +366,13 @@ copyDependencyToDir <- function(dependency, outputDir, mustWork = TRUE) {
       )
     )
   }
+  if (!length(srcfiles)) {
```

Comment:
May we also move this if-block up to just after `files` is created? (~359)

```suggestion
  if (length(files) == 0) {
```

Reply:
Done in d945eb5!